src/service/icon_generator/icon_generator.service.ts

```diff
@@ -0,0 +1,116 @@
+'use strict';
+import * as Jimp from 'jimp';
+import * as crypto from 'crypto';
+import { Complex } from 'src/shared/util/mathUtils';
+
+const imageSize = 1028;
+const halfSize = imageSize / 2;
+
+const colorElementNumber = 4;
+const emptyColor = [240, 240, 240, 255];
+
+const colorRange = 256;
+const minColor = 0;
+const maxColor = colorRange - 1;
+
+const minColorIntensity = 70;
+const maxColorIntensity = 230;
+const magicRandomNumber = 10005;
+
+const juliaSetThreshold = 10;
+const juliaSetMaxIter = 20;
+
+const hashDigitsToAccount = 10;
+const hex = 16;
+
+const scaleMultiplier = 2 / imageSize;
```

Comment:
This should be put in a separate file `shared/const/AvatarGen.ts`
And maybe for easy import you should use this syntax
```javascript
export const AvatarGenConstants = Object.freeze({
  imageSize: 1028,
  halfSize: 512,
  ...
});
```
Or this:
```javascript
const imageSize = 1028;
const halfSize = imageSize / 2;
...

export default Object.freeze({
  imageSize,
  halfSize,
  ...
});
```

Reply:
Or add new private readonly variable to the class